File: Source/OregonTilth.Web/src/app/shared/models/forms/transplant-production-information/transplant-production-information-create-dto.ts
export class TransplantProductionInformationCreateDto {
    WorkbookID: number;
    CropID: number;
    PhaseID: number;
    TransplantProductionTrayTypeID: number;
    SeedsPerTray: number;
    UsageRate: number;
    CostPerSeed: number;
    CropSpecificInputCostsPerTray: number;
    constructor(obj?: any) {
        Object.assign(this, obj);
    }
}

File: OregonTilth.Models/DataTransferObjects/Workbook/Forms/TransplantProductionInformation/TransplantProductionInformationCreateDto.cs
using System.ComponentModel.DataAnnotations;

namespace OregonTilth.Models.DataTransferObjects
{
    public class TransplantProductionInformationCreateDto
    {
        [Required] public int WorkbookID { get; set; }
        [Required] public int CropID { get; set; }
        [Required] public int PhaseID { get; set; }
        [Required] public int TransplantProductionTrayTypeID { get; set; }
        [Required] public int SeedsPerTray { get; set; }
        [Required] public decimal UsageRate { get; set; }
        public decimal CostPerSeed { get; set; }
        public decimal CropSpecificInputCostsPerTray { get; set; }
    }
}